### PR TITLE
feat(breadcrumb): webcomponent

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Web component version of the breadcrumb.

**Solving issue**  
Fixes: [AB#2854](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2854)

**How to test**  
1. Go to storybook link below
2. Check in Components -> Breadcrumb -> WebComponent

**Screenshots**  
-

**Additional context**  
-

### CODE REVIEW

File: tegel/src/components/breadcrumb/breadcrumb-item/breadcrumb-item.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sdds-breadcrumb-item',
+  styleUrl: 'breadcrumb-item.scss',
+  shadow: true,
+})
+export class SddsBreadcrumb {
```
Comment (mJarsater) [inline tegel/src/components/breadcrumb/breadcrumb-item/breadcrumb-item.tsx:8]:
```suggestion
export class SddsBreadcrumbItem {
```

**Comment by EmelieLitwin** [comment]
Yeay! Making web components 😍 
Some thoughts:
Maybe the breadcrumb could be a nav-element instead of host and have the slots in an ol for WCAG reasons:
https://www.w3.org/WAI/ARIA/apg/example-index/breadcrumb/index.html
I don't know if we want to get a head start on a bit of a11y for new components, but there is also an aria state called aria-current.

**Comment by mJarsater** [comment]
> Yeay! Making web components 😍 Some thoughts: Maybe the breadcrumb could be a nav-element instead of host and have the slots in an ol for WCAG reasons: https://www.w3.org/WAI/ARIA/apg/example-index/breadcrumb/index.html I don't know if we want to get a head start on a bit of a11y for new components, but there is also an aria state called aria-current.

Great idea! I'll update the PR 😄

**Comment by mJarsater** [comment]
@EmelieLitwin updated the PR now. Please have a look :)

**Comment by EmelieLitwin** [comment]
> @EmelieLitwin updated the PR now. Please have a look :)

Nice! :) I was thinking that the disabled breadcrumb in the storybook story might not be needed. It looks a little off and hopefully not a common use case.

I also saw in figma that hovered breadcrumb is supposed to be underlined.

Otherwise, looks good 🥳

File: tegel/src/components/breadcrumb/breadcrumb-component.stories.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      {
+        name: 'Link',
+        type: 'link',
+        url: 'https://www.figma.com/file/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=2703%3A4725&t=rVXuTOgTmXPauyHd-1',
+      },
+    ],
+  },
+};
+
+const Template = () => `
```
Comment (theJohnnyMe) [inline tegel/src/components/breadcrumb/breadcrumb-component.stories.ts:19]: Please add formatHTML here.

File: tegel/src/components/breadcrumb/breadcrumb-component.stories.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+export default {
```
Comment (theJohnnyMe) [inline tegel/src/components/breadcrumb/breadcrumb-component.stories.ts:1]: Please add the breadcrumb item readme to the "Notes" tab of Storybook.

**Review by theJohnnyMe — APPROVED** [review]
Please rename the "Default" story to "Native" just as that is how we named it in other components that exist in both HTML&CSS and WebComponent versions. 

Other than that tiny things, it looks good!

**Comment by mJarsater** [comment]
> Please rename the "Default" story to "Native" just as that is how we named it in other components that exist in both HTML&CSS and WebComponent versions.
> 
> Other than that tiny things, it looks good!

Nice catch!

**Comment by mJarsater** [comment]
@theJohnnyMe please have a look again. Removed the link-text prop and used a slot instead, I think it looks a bit neater 😄

**Comment by theJohnnyMe** [comment]
> @theJohnnyMe please have a look again. Removed the link-text prop and used a slot instead, I think it looks a bit neater 😄

That was a good call really, then component reminds more of the native anchor tag + it enables uses to pass HTML in too, in case they want to have an SVG icon text to text too or something similar.

**Pull request merged**